packages/component/src/SendBox/ConnectivityStatus.js

```diff
@@ -54,6 +56,12 @@ class connectConnectivityAlert extends React.Component {
     }
   }
 
+  dispatchConnectivityTimeout() {
+    setTimeout( () => {
```

Comment:
this should return the call to `setTimeout()` so that the caller of `dispatchConnectivityTimeout()` can receive the timer id.

Ex.

```
dispatchConnectivityTimeout() {
  return setTimeout(() => 
    ...
```